osquery/distributed/distributed.cpp

```diff
@@ -171,14 +171,37 @@ Status Distributed::acceptWork(const std::string& work) {
       std::stringstream ss(work);
       pt::read_json(ss, tree);
     }
+    std::set<std::string> queries_to_run;
+    // Check for and run discovery queries first
+    if (tree.count("discovery") > 0) {
+      auto& queries = tree.get_child("discovery");
+
+      for (const auto& node : queries) {
+        auto query = queries.get<std::string>(node.first, "");
+        if (query.empty() || node.first.empty()) {
+          return Status(
+              1,
+              "Distributed discovery query does not have complete attributes");
+        }
+        SQL sql(query);
+        if (!sql.getStatus().ok()) {
+          return Status(1, "Distributed discovery query has an SQL error");
```

Comment:
Depends on the pronunciation. If we want to adopt "sequel" then "a", if we want to adopt "es-queue-ell", then "an". Which ever we want to make the standard I'm good with.